openghg/retrieve/_access.py

```diff
@@ -769,9 +769,13 @@ def _metadata_difference(
         if print_output:
             logger.info(f" {param}: ")
         for m in metadata:
-            summary_difference[param].append(m[param])
+            if param in m:
+                value: Optional[str] = m[param]
```

Comment:
Can this be a `None`?

Reply:
It guess it depends on if it's possible for the parameter in the metadata can be None? Potentially not.

Reply:
This has been updated to remove Optional[]